atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -275,19 +276,22 @@ def run(self):
         """
         run the plugin
         """
-        try:
-            config = self.workflow.base_image_inspect[INSPECT_CONFIG]
-        except (AttributeError, TypeError):
-            message = "base image was not inspected"
-            self.log.error(message)
-            raise RuntimeError(message)
-        else:
-            base_image_labels = config["Labels"] or {}
-
         dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
 
         lines = dockerfile.lines
 
+        if re.match('^koji/image-build(:.*)?$', dockerfile.baseimage):
```

Comment:
Looking at the code, `KeyError` is only raised if `docker.errors.NotFound` is raised.
Any other errors should propagate, causing a build failure.